pkg/addons/data/ConfigMap/openshift-metrics/prometheus.yaml

```diff
@@ -50,32 +52,32 @@ data:
     # Each master node exposes a /metrics endpoint on :8444 that contains operational metrics for
     # the controllers.
     #
-    - job_name: 'kubernetes-controllers'
-
-      scheme: https
-      tls_config:
-        ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
-      bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
-
-      kubernetes_sd_configs:
-      - role: endpoints
-        namespaces:
-          names:
-          - default
-
-      # Keep only the default/kubernetes service endpoints for the https port, and then
-      # set the port to 8444. This is the default configuration for the controllers on OpenShift
-      # masters.
-      relabel_configs:
-      - source_labels: [__meta_kubernetes_service_name, __meta_kubernetes_endpoint_port_name]
-        action: keep
-        regex: kubernetes;https
-      - source_labels: [__address__]
-        action: replace
-        target_label: __address__
-        regex: (.+)(?::\d+)
-        replacement: $1:8444
-
+    #- job_name: 'kubernetes-controllers'
+    #
+    #  scheme: https
+    #  tls_config:
+    #    ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+    #  bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
+    #
+    #  kubernetes_sd_configs:
+    #  - role: endpoints
+    #    namespaces:
+    #      names:
+    #      - default
+    #
+    #  # Keep only the default/kubernetes service endpoints for the https port, and then
+    #  # set the port to 8444. This is the default configuration for the controllers on OpenShift
+    #  # masters.
+    #  relabel_configs:
+    #  - source_labels: [__meta_kubernetes_service_name, __meta_kubernetes_endpoint_port_name]
+    #    action: keep
+    #    regex: kubernetes;https
+    #  - source_labels: [__address__]
+    #    action: replace
+    #    target_label: __address__
+    #    regex: (.+)(?::\d+)
+    #    replacement: $1:8444
+    #
```

Comment:
don't forget to open an issue to fix this

Reply:
will do when squashing :)

Reply:
we tend to open the issue then reference it here in the review, that way it doesn't get dropped

Reply:
https://github.com/jim-minter/azure-helm/issues/100